sql/updates/update_2024-08-11_1.sql

```diff
@@ -1,18 +1,17 @@
-CREATE TABLE `character_etc_properties` (
-  `propertyId` bigint(20) NOT NULL,
+CREATE TABLE IF NOT EXISTS `character_etc_properties` (
```

Comment:
Can you tell me what this change is about? I feel like we'd want this to fail if the table already exists, because that's an indicator for something not being right.